tests/test_edgeql_select.py

```diff
@@ -6288,6 +6288,34 @@ async def test_edgeql_select_big_set_literal(self):
 
         assert len(res) == 100
 
+    async def test_edgeql_select_big_unions(self):
+        res = await self.con.query("""
+            SELECT (
+                 (1,) union (1,) union (1,) union (1,) union (1,) union
```

Comment:
Can you add a similar test with `{1, 2, 3, 4, 5, 6, ..., 100}`? That used to crash with a recursive error.

Reply:
We already have a test for that.